.github/workflows/gating.yaml

```diff
@@ -116,4 +116,5 @@ jobs:
           CACHI2_IMAGE: localhost/cachi2:${{ github.sha }}
           CACHI2_TEST_LOCAL_PYPISERVER: 'true'
         run: |
+          git config --global --add safe.directory "*"
```

Comment:
**Commit message:**

nitpick: Can we be more explicit in the subject on where exactly the problem is? I know you say 'gate', but it may not be obvious to everyone, so either change the context or just mention GitHub CI somewhere in both the subject AND/OR the commit message itself.

Reply:
Done.